tests/dir-ext.rs

```diff
@@ -11,7 +11,7 @@ fn remove_file() {
     let file = tempdir.create("file").expect("create file to delete");
     drop(file);
     tempdir.remove_file_or_symlink("file").expect("delete file");
-    assert!(!tempdir.exists("file"), "deletion worked");
+    assert!(!tempdir.try_exists("file").unwrap(), "deletion worked");
```

Comment:
Can we leave the `exists` test here, and have both `exists` and `try_exists` tests here?